[2024-11-21 16:24:26,714] INFO - musicbot.launcher | In run.py::MainThread(12096), line 858 in main: Loading MusicBot version:  release-040724-dirty
[2024-11-21 16:24:26,717] INFO - musicbot.launcher | In run.py::MainThread(12096), line 859 in main: Log opened:  Thu Nov 21 16:24:26 2024
[2024-11-21 16:24:26,717] INFO - musicbot.launcher | In run.py::MainThread(12096), line 337 in sanity_checks: Starting sanity checks
[2024-11-21 16:24:26,717] INFO - musicbot.launcher | In run.py::MainThread(12096), line 374 in req_ensure_py3: Checking for Python 3.8+
[2024-11-21 16:24:26,717] INFO - musicbot.launcher | In run.py::MainThread(12096), line 434 in req_ensure_py3: Python version:  3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)]
[2024-11-21 16:24:26,717] INFO - musicbot.launcher | In run.py::MainThread(12096), line 463 in req_ensure_env: Ensuring we're in the right environment
[2024-11-21 16:24:26,718] INFO - musicbot.launcher | In run.py::MainThread(12096), line 507 in req_ensure_env: Adding local bins/ folder to path
[2024-11-21 16:24:26,921] INFO - musicbot.launcher | In run.py::MainThread(12096), line 351 in sanity_checks: Required checks passed.
[2024-11-21 16:24:26,922] INFO - musicbot.launcher | In run.py::MainThread(12096), line 364 in sanity_checks: Skipped checking for updates.
[2024-11-21 16:24:26,922] INFO - musicbot.launcher | In run.py::MainThread(12096), line 366 in sanity_checks: Optional checks passed.
[2024-11-21 16:24:27,893] DEBUG - musicbot.entry | In entry.py::MainThread(12096), line 43 in <module>: module 'pymediainfo' not found, will fall back to ffprobe.
[2024-11-21 16:24:27,904] INFO - musicbot.bot | In bot.py::MainThread(12096), line 129 in __init__: Initializing MusicBot release-040724-dirty
[2024-11-21 16:24:27,909] INFO - musicbot.config | In config.py::MainThread(12096), line 103 in __init__: Loading config from:  config\options.ini
[2024-11-21 16:24:27,912] INFO - musicbot.utils | In utils.py::MainThread(12096), line 238 in set_logging_level: Changing log level to:  INFO
[2024-11-21 16:24:27,914] INFO - musicbot.config | In config.py::MainThread(12096), line 836 in run_checks: Using i18n: config\i18n\en.json
[2024-11-21 16:24:27,915] INFO - musicbot.config | In config.py::MainThread(12096), line 869 in run_checks: Audio Cache will be stored in:  D:\GITHUB\MusicBot\MusicBot\audio_cache
[2024-11-21 16:24:29,194] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 238 in setup_hook: The config did not have Spotify app credentials, attempting to use guest mode.
[2024-11-21 16:24:29,499] INFO - musicbot.bot | In bot.py::MainThread(12096), line 248 in setup_hook: Authenticated with Spotify successfully using guest mode.
[2024-11-21 16:24:29,499] INFO - musicbot.bot | In bot.py::MainThread(12096), line 258 in setup_hook: Initialized, now connecting to discord.
[2024-11-21 16:24:32,954] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2052 in _on_ready_once: MusicBot:  1094024367886831617/Bot của Toàn#7653
[2024-11-21 16:24:32,954] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2061 in _on_ready_once: Owner:     753272101343068160/tranduytoan#0

[2024-11-21 16:24:32,955] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2068 in _on_ready_once: Guild List:
[2024-11-21 16:24:32,955] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2072 in _on_ready_once:  - Shibe on top
[2024-11-21 16:24:32,955] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2159 in _on_ready_once: Not bound to any text channels
[2024-11-21 16:24:32,955] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2208 in _on_ready_once: Not autojoining any voice channels
[2024-11-21 16:24:36,951] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 16:24:36,952] INFO - musicbot.bot | In bot.py::MainThread(12096), line 518 in _auto_join_channels: Found owner in voice channel:  lol
[2024-11-21 16:24:36,952] INFO - musicbot.bot | In bot.py::MainThread(12096), line 550 in _auto_join_channels: Attempting to join channel:  Shibe on top/lol  in guild:  Shibe on top
[2024-11-21 16:24:38,318] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 16:24:38,320] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=WL11bwvAYWI
[2024-11-21 16:24:43,232] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=WL11bwvAYWI
[2024-11-21 16:24:56,128] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7188 in on_message: Message from 753272101343068160/tranduytoan: ?skip f
[2024-11-21 16:24:56,184] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-WL11bwvAYWI-NGAY_MAI_NG_I_TA_L_Y_CH_NG_-_THANH_T_x_ONG_THIEN_C_OFFICIAL_MV_Ngay_Hom_y_Em_i_Trong_M_a-NA.webm'
[2024-11-21 16:24:56,185] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-WL11bwvAYWI-NGAY_MAI_NG_I_TA_L_Y_CH_NG_-_THANH_T_x_ONG_THIEN_C_OFFICIAL_MV_Ngay_Hom_y_Em_i_Trong_M_a-NA.webm'
[2024-11-21 16:24:56,185] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-WL11bwvAYWI-NGAY_MAI_NG_I_TA_L_Y_CH_NG_-_THANH_T_x_ONG_THIEN_C_OFFICIAL_MV_Ngay_Hom_y_Em_i_Trong_M_a-NA.webm'
[2024-11-21 16:24:59,278] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=fB8TyLTD7EE
[2024-11-21 16:25:02,515] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=fB8TyLTD7EE
[2024-11-21 16:28:35,817] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=C5fLxtJH2Qs
[2024-11-21 16:28:39,063] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=C5fLxtJH2Qs
[2024-11-21 16:32:25,793] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=1Z6CHioIn3s
[2024-11-21 16:32:28,890] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=1Z6CHioIn3s
[2024-11-21 16:36:03,858] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=Zasx9hjo4WY
[2024-11-21 16:36:07,042] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=Zasx9hjo4WY
[2024-11-21 16:39:56,082] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=L7kF4MXXCoA
[2024-11-21 16:39:59,441] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=L7kF4MXXCoA
[2024-11-21 16:43:37,474] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=C3GouGa0noM
[2024-11-21 16:43:40,648] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=C3GouGa0noM
[2024-11-21 16:47:58,958] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=L-VMCv4Y9l8
[2024-11-21 16:48:02,067] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=L-VMCv4Y9l8
[2024-11-21 16:51:49,013] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=Nqqc2FHf9Ug
[2024-11-21 16:51:52,520] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=Nqqc2FHf9Ug
[2024-11-21 16:56:47,312] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=PebLShakKCE
[2024-11-21 16:56:50,677] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=PebLShakKCE
[2024-11-21 17:01:33,856] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=liPu1_aPH5k
[2024-11-21 17:01:37,016] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=liPu1_aPH5k
[2024-11-21 17:05:13,595] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=eEFVxI9lqjU
[2024-11-21 17:05:16,771] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=eEFVxI9lqjU
[2024-11-21 17:07:58,756] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=ZHhqwBwmRkI
[2024-11-21 17:08:02,027] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=ZHhqwBwmRkI
[2024-11-21 17:12:24,052] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 17:12:26,505] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 17:16:49,829] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=CdZN8PI3MqM
[2024-11-21 17:16:53,357] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=CdZN8PI3MqM
[2024-11-21 17:20:20,533] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=S19UcWdOA-I
[2024-11-21 17:20:23,903] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=S19UcWdOA-I
[2024-11-21 17:23:41,179] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=UOxkGD8qRB4
[2024-11-21 17:23:44,637] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=UOxkGD8qRB4
[2024-11-21 17:27:10,730] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=U9pGr6KMdyg
[2024-11-21 17:27:14,183] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=U9pGr6KMdyg
[2024-11-21 17:30:59,369] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=aR-KAldshAE
[2024-11-21 17:31:02,803] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=aR-KAldshAE
[2024-11-21 17:34:27,849] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=kRauptdx_Is
[2024-11-21 17:34:31,449] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=kRauptdx_Is
[2024-11-21 17:37:48,075] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=dj4VoPO-2pE
[2024-11-21 17:37:50,324] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=dj4VoPO-2pE
[2024-11-21 17:41:47,761] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=r6zIGXun57U
[2024-11-21 17:41:51,126] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=r6zIGXun57U
[2024-11-21 17:44:53,133] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=i1IKnWDecwA
[2024-11-21 17:44:57,060] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=i1IKnWDecwA
[2024-11-21 17:48:28,282] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 17:48:31,963] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 17:52:03,765] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=3VTkBuxU4yk
[2024-11-21 17:52:07,162] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=3VTkBuxU4yk
[2024-11-21 17:52:35,626] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 17:52:36,423] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 17:52:36,424] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 17:52:36,424] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 17:52:36,424] INFO - musicbot.bot | In bot.py::MainThread(12096), line 540 in _auto_join_channels: Already connected to channel:  lol  in guild:  Shibe on top
[2024-11-21 17:52:36,425] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 17:56:01,256] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=3S1NmtWDVYk
[2024-11-21 17:56:04,655] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=3S1NmtWDVYk
[2024-11-21 18:00:23,506] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=i1IKnWDecwA
[2024-11-21 18:00:26,844] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=i1IKnWDecwA
[2024-11-21 18:03:57,952] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=eEFVxI9lqjU
[2024-11-21 18:04:01,537] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=eEFVxI9lqjU
[2024-11-21 18:06:44,485] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=Nqqc2FHf9Ug
[2024-11-21 18:06:48,086] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=Nqqc2FHf9Ug
[2024-11-21 18:11:42,510] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=ZHhqwBwmRkI
[2024-11-21 18:11:45,452] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=ZHhqwBwmRkI
[2024-11-21 18:16:08,013] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=r6zIGXun57U
[2024-11-21 18:16:10,777] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=r6zIGXun57U
[2024-11-21 18:19:12,622] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=kRauptdx_Is
[2024-11-21 18:19:15,553] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=kRauptdx_Is
[2024-11-21 18:22:31,917] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 18:22:34,099] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 18:26:57,189] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=aR-KAldshAE
[2024-11-21 18:27:00,002] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=aR-KAldshAE
[2024-11-21 18:30:24,438] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=3S1NmtWDVYk
[2024-11-21 18:30:27,480] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=3S1NmtWDVYk
[2024-11-21 18:34:45,985] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=3VTkBuxU4yk
[2024-11-21 18:34:49,044] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=3VTkBuxU4yk
[2024-11-21 18:35:47,189] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 18:35:48,121] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 18:35:48,122] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 18:35:48,122] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 18:35:48,123] INFO - musicbot.bot | In bot.py::MainThread(12096), line 540 in _auto_join_channels: Already connected to channel:  lol  in guild:  Shibe on top
[2024-11-21 18:35:48,123] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 18:38:42,798] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=CdZN8PI3MqM
[2024-11-21 18:38:45,763] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=CdZN8PI3MqM
[2024-11-21 18:42:12,274] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=UOxkGD8qRB4
[2024-11-21 18:42:15,249] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=UOxkGD8qRB4
[2024-11-21 18:45:40,935] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=S19UcWdOA-I
[2024-11-21 18:45:44,003] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=S19UcWdOA-I
[2024-11-21 18:49:00,726] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=Zasx9hjo4WY
[2024-11-21 18:49:03,731] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=Zasx9hjo4WY
[2024-11-21 18:52:52,808] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 18:52:55,934] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 18:56:27,149] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=liPu1_aPH5k
[2024-11-21 18:56:30,195] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=liPu1_aPH5k
[2024-11-21 19:00:06,909] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=U9pGr6KMdyg
[2024-11-21 19:00:10,092] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=U9pGr6KMdyg
[2024-11-21 19:03:39,998] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:03:40,933] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 19:03:40,933] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 19:03:40,933] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 19:03:40,933] INFO - musicbot.bot | In bot.py::MainThread(12096), line 540 in _auto_join_channels: Already connected to channel:  lol  in guild:  Shibe on top
[2024-11-21 19:03:40,933] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 19:03:54,780] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=L-VMCv4Y9l8
[2024-11-21 19:03:57,861] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=L-VMCv4Y9l8
[2024-11-21 19:07:44,818] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=C3GouGa0noM
[2024-11-21 19:07:48,209] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=C3GouGa0noM
[2024-11-21 19:12:05,602] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=dj4VoPO-2pE
[2024-11-21 19:12:07,947] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=dj4VoPO-2pE
[2024-11-21 19:16:05,060] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=fB8TyLTD7EE
[2024-11-21 19:16:08,021] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=fB8TyLTD7EE
[2024-11-21 19:19:41,020] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=C5fLxtJH2Qs
[2024-11-21 19:19:44,428] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=C5fLxtJH2Qs
[2024-11-21 19:20:36,920] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:20:37,694] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 19:20:37,694] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 19:20:37,694] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 19:20:37,694] INFO - musicbot.bot | In bot.py::MainThread(12096), line 540 in _auto_join_channels: Already connected to channel:  lol  in guild:  Shibe on top
[2024-11-21 19:20:37,694] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 19:21:03,706] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:21:03,765] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:21:03,924] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:21:04,729] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:21:09,990] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:21:09,991] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:21:09,991] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 550 in _auto_join_channels: Attempting to join channel:  Shibe on top/lol  in guild:  Shibe on top
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 560 in _auto_join_channels: Discarding MusicPlayer and making a new one...
[2024-11-21 19:21:12,019] INFO - musicbot.bot | In bot.py::MainThread(12096), line 782 in disconnect_voice_client: Disconnecting a MusicPlayer in guild:  Shibe on top
[2024-11-21 19:21:26,026] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 750 in get_voice_client: Retrying connection after a timeout error (1) while trying to connect to:  lol
[2024-11-21 19:21:28,250] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 19:23:40,854] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:23:40,876] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:23:50,868] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:23:50,868] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:23:50,871] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-C5fLxtJH2Qs-Egzod_Maestro_Chives_-_Royalty_ft._Neoni_Trap_NCS_-_Copyright_Free_Music-NA.webm'
[2024-11-21 19:23:54,830] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 2017 in on_resumed: MusicBot resumed a session with discord.
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 490 in _auto_join_channels: Found resumable voice channel:  lol  in guild:  Shibe on top
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 550 in _auto_join_channels: Attempting to join channel:  Shibe on top/lol  in guild:  Shibe on top
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 560 in _auto_join_channels: Discarding MusicPlayer and making a new one...
[2024-11-21 19:24:10,300] INFO - musicbot.bot | In bot.py::MainThread(12096), line 782 in disconnect_voice_client: Disconnecting a MusicPlayer in guild:  Shibe on top
[2024-11-21 19:24:24,318] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 750 in get_voice_client: Retrying connection after a timeout error (1) while trying to connect to:  lol
[2024-11-21 19:24:25,270] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 19:28:04,623] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 19:28:07,730] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=EIwtr-VqwcM
[2024-11-21 19:32:33,503] INFO - musicbot.entry | In entry.py::MainThread(12096), line 667 in _really_download: Download started:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 19:32:39,812] INFO - musicbot.entry | In entry.py::MainThread(12096), line 699 in _really_download: Download complete:  https://www.youtube.com/watch?v=AnMhdn0wJ4I
[2024-11-21 19:39:04,149] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:39:04,339] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:39:15,504] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-AnMhdn0wJ4I-Vicetone_-_Nevada_ft._Cozi_Zuehlsdorff-NA.webm'
[2024-11-21 19:39:15,504] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-AnMhdn0wJ4I-Vicetone_-_Nevada_ft._Cozi_Zuehlsdorff-NA.webm'
[2024-11-21 19:39:15,504] WARNING - musicbot.player | In player.py::MainThread(12096), line 465 in _handle_file_cleanup: Cannot delete file due to a permissions error.
Traceback (most recent call last):
  File "D:\GITHUB\MusicBot\MusicBot\musicbot\player.py", line 458, in _handle_file_cleanup
    os.unlink(filename)
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'D:\\GITHUB\\MusicBot\\MusicBot\\audio_cache\\youtube-AnMhdn0wJ4I-Vicetone_-_Nevada_ft._Cozi_Zuehlsdorff-NA.webm'
[2024-11-21 19:39:22,943] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:40:26,711] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7502 in on_disconnect: MusicBot has become disconnected.
[2024-11-21 19:40:32,524] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7498 in on_connect: MusicBot has become connected.
[2024-11-21 19:40:34,552] INFO - musicbot.bot | In bot.py::MainThread(12096), line 7773 in on_guild_available: Guild "Shibe on top" has become available.
[2024-11-21 19:40:38,553] INFO - musicbot.bot | In bot.py::MainThread(12096), line 470 in _auto_join_channels: Checking for channels to auto-join or resume...
[2024-11-21 19:40:38,553] INFO - musicbot.bot | In bot.py::MainThread(12096), line 605 in _auto_join_channels: Finished joining configured channels.
[2024-11-21 19:55:58,333] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 3 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:56:01,325] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 6 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:56:07,332] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 9 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:56:16,343] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 12 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:56:28,345] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 15 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:56:43,358] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 18 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:57:01,359] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 21 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:57:22,357] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 24 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:57:46,375] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 27 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:58:13,381] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 30 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:58:43,388] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 33 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:59:16,397] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 36 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 19:59:52,397] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: VoiceClient not connected, waiting 39 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:00:31,388] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 42 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:01:13,411] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 45 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:01:58,421] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 48 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:02:46,435] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 51 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:03:37,459] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 54 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:04:31,462] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 57 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:05:28,480] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 60 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:06:28,493] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 63 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:07:31,489] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 66 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:08:37,504] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 69 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:09:46,500] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 72 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:10:58,513] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 75 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:12:13,516] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 78 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:13:31,532] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 81 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:14:52,545] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 84 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:16:16,567] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 87 seconds to handle auto-pause in guild:  Shibe on top
[2024-11-21 20:17:43,574] WARNING - musicbot.bot | In bot.py::MainThread(12096), line 3178 in _handle_guild_auto_pause: [Bug] VoiceClient not connected, waiting 90 seconds to handle auto-pause in guild:  Shibe on top
